src/component/term/TermDetail.tsx

```diff
@@ -93,7 +93,7 @@ export class TermDetail extends EditableComponent<TermDetailProps> {
         const buttons = this.getButtons();
         return <div id="term-detail">
             <HeaderWithActions title={
-                <>{term.label}<CopyIriIcon url={term.iri as string}/></>
+                <>{term.label}<CopyIriIcon url={term.iri as string}/><br/><h6>{Utils.sanitizeArray(term.altLabels || []).join(", ")}</h6></>
```

Comment:
Calling `Utils.sanitizeArray(term.altLabels || [])` is superfluous, sanitization ensures that an array is returned, so there's no need for the conditional argument.

Reply:
fixed